src/components/30-organisms/table-sortable/story.js

```diff
@@ -0,0 +1,73 @@
+import { storiesOf } from '@storybook/html';
+import './index';
+
+import { withMarkdown } from '../../../../.storybook/addons/markdown';
+import Readme from './README.md';
+
+const model = {
+  thead: [
+    { html: 'Title 0', sort: 'ASC' },
+    { html: 'Title 1', sort: 'ASC' },
+    { html: 'Title 3', sort: 'DESC' },
+    { html: 'Title 2' },
+  ],
+  tbody: [
+    [
+      { html: '<span>11 Gaa</span>' },
```

Comment:
Please use better "test" or even real case test strings in cells. (Lipsum, names..., longer, shorter)

Reply:
ok i will

Reply:
done